example/client/cmd/report.go

```diff
@@ -15,17 +15,77 @@
 package cmd
 
 import (
+	"context"
+	"fmt"
+	"io"
+	"time"
+
+	"github.com/golang/protobuf/ptypes"
```

Comment:
I think you need to add this dependency to the BUILD file (based on the Travis errors).